CHANGELOG.MD

```diff
@@ -1,3 +1,22 @@
+## 1.289.1 (2024-04-17)
+
+### Features
```

Comment:
These are not features, I would call these "Fixes" and mention the trigger but also remark that we are not actually affected.
Maybe a single point "Update dependency versions" would be enough? Details can be seen in the diffs anyways if needed.

Reply:
I changed this, please check again